settings/common.py

```diff
@@ -61,6 +61,7 @@
 THIRD_PARTY_APPS = [
     "allauth",
     "allauth.account",
+    'allauth.socialaccount',
```

Comment:
Why are we adding this app?

Reply:
After upgrading the allauth package we had to add this app explicitly to get it working. I can't find the git issue where I found the solution. I will update if I find it here